### ISSUE TITLE
mimic joint support

### ISSUE THREAD

**Issue by gonultasbu** [nimblephysics/nimblephysics#83]
Hello,

Does the project support mimic joints? It looks like the loader is able to load an .urdf model with mimic joints but the backpropagation terminates without raising error if the .urdf model contains a mimic joint after I run `loss.backward()`.
